index.js

```diff
@@ -5,10 +5,10 @@ module.exports = function (rgbString) {
 	// First remove all whitespace, then parse the rgb string
 	rgbString = rgbString.replace(/ /g, '');
 
-	rgbString.replace(/rgb\((\d+),(\d+),(\d+)\)/, function (rgbString, r, g, b) {
-		rgb.red = Number(r);
-		rgb.green = Number(g);
-		rgb.blue = Number(b);
+	rgbString.replace(/rgb\((\d+|-\d+),(\d+|-\d+),(\d+|-\d+)\)/, function (rgbString, r, g, b) {
+		rgb.red = r > 0 ? Number(r) : 0;
```

Comment:
thinking about this more, let's just parse it to the actual negative number. Can you update you PR?